kong/plugins/aws-lambda/handler.lua

```diff
@@ -51,6 +52,17 @@ function AWSLambdaHandler:access(conf)
     query = conf.qualifier and "Qualifier=" .. conf.qualifier
   }
 
+  if conf.aws_iam_role then
+    local credentials, err = aws_metadata.credentials(conf.aws_iam_role)
```

Comment:
It assigns the first return value to `credentials` and the [optional] second to `err`. If `err` was not present then `nil`.